src/tests/Directory.Build.targets

```diff
@@ -542,6 +542,7 @@
     <IlcReference Include="$(TargetingPackPath)/*.dll" />
   </ItemGroup>
 
+  <Import Project="$(CoreCLRBuildIntegrationDir)Microsoft.DotNet.ILCompiler.SingleEntry.targets" Condition="'$(TestBuildMode)' == 'nativeaot'" />
   <Import Project="$(CoreCLRBuildIntegrationDir)Microsoft.NETCore.Native.targets" Condition="'$(TestBuildMode)' == 'nativeaot'" />
```

Comment:
```suggestion
```
SingleEntry.targets imports Microsoft.NETCore.Native.targets so this import should not be necessary.